pkg/reconciler/intevents/pullsubscription/resources/receive_adapter.go

```diff
@@ -138,9 +128,6 @@ func makeReceiveAdapterPodSpec(ctx context.Context, args *ReceiveAdapterArgs) *c
 		}, {
 			Name:  "ADAPTER_TYPE",
 			Value: adapterType,
-		}, {
-			Name:  "SEND_MODE",
-			Value: string(mode),
```

Comment:
did you also remove this env variable from the adapter code?

Reply:
Which part do you mean?

Reply:
I searched for "SEND_MODE", I can't find any usage of it in the codebase.